<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>backpackMan</class>
 <widget class="QDialog" name="backpackMan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="columnCount">
      <number>2</number>
     </property>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="refreshPage">
     <property name="text">
      <string>获取页面</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="previousPage">
     <property name="text">
      <string>上一页</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sell">
     <property name="text">
      <string>卖出</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextPage">
     <property name="text">
      <string>下一页</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
